lib/wanda/checks_customizations.ex

```diff
@@ -14,6 +16,15 @@ defmodule Wanda.ChecksCustomizations do
           value: any()
         }
 
+  @spec get_customizations(Ecto.UUID.t()) :: [CheckCustomization.t()]
+  def get_customizations(group_id) do
+    Repo.all(
+      from c in CheckCustomization,
+        where: c.group_id == ^group_id,
+        select: c
```

Comment:
Do we need this `select`?

Reply:
Thanks! da886aa77e3f903e1e949da32796d610ead3a35a